bodyfetcher.py

```diff
@@ -80,9 +80,6 @@ def make_api_call_for_site(self, site):
             if response["quota_remaining"] - GlobalVars.apiquota >= 1000 and GlobalVars.apiquota >= 0:
                 GlobalVars.charcoal_hq.send_message("API quota rolled over with {} requests remaining.".format(GlobalVars.apiquota))
             GlobalVars.apiquota = response["quota_remaining"]
-        else:
-            GlobalVars.apiquota = 0
-            return
```

Comment:
The lack of return doesn't make a difference in the end; if there is an error then we don't get [items] from the server, and that's the end of the function.